proto/rill/admin/v1/api.proto

```diff
@@ -281,6 +281,7 @@ message GetGithubRepoStatusResponse {
   bool has_access = 1;
   string grant_access_url = 2;
   string default_branch = 3;
+  string user_authorisation_url = 4;
```

Comment:
Is the different messaging in the CLI really necessary? In either case, it seems to me we could just say "Open browser to grant access to Github" (user doesn't need to know which kind, feels like an implementation detail)

Reply:
It will be weird to show `grant access to Github` when they just asked admin to grant access. It may be okay if we continue to auto open browser though.